phone/src/apps/messages/components/modal/MessageModal.tsx

```diff
@@ -176,11 +151,11 @@ export const MessageModal = () => {
             </Typography>
             <Tooltip
               classes={{ tooltip: classes.tooltip }}
-              title={t('MESSAGES.ACTIONS_TITLE')}
-              placement="left"
+              title={`${t('DIALER.NAVBAR_DIAL')} ${targetNumber}`}
```

Comment:
Let's use the generic translation key here :)

```suggestion
              title={`${t('GENERIC.CALL')} ${targetNumber}`}
```

Reply:
Ah I was searching for it but couldn't find it.